tests/finite_element_problem/test_linear_simple_beam.py

```diff
@@ -0,0 +1,64 @@
+import os
+import pytest
+from pathlib import Path
+
+from fenicsxconcrete.helper import Parameters
+from fenicsxconcrete.unit_registry import ureg
+from fenicsxconcrete.experimental_setup.simple_beam import SimpleBeam
+from fenicsxconcrete.finite_element_problem.linear_elasticity import LinearElasticity
+from fenicsxconcrete.sensor_definition.displacement_sensor import DisplacementSensor
+
+
+@pytest.mark.parametrize("dimension", [[2, [0.00105057, -0.01310806]],
```

Comment:
Where does the results come from? an analytic solution or an fem convergence study?

Reply:
as we discussed, this is an arbitrary result, trusting in the model and just checking that the result does not change.